src/main/resources/logback-spring.xml

```diff
@@ -45,6 +45,10 @@
             <appender-ref ref="consoleAppender"/>
         </logger>
 
+        <logger name="org.flywaydb.core.internal.sqlscript.DefaultSqlScriptExecutor" additivity="false" level="WARN">
```

Comment:
these were annoying, change recently to flyway means that every sql script posts a count of rows affected, which for community api is lots